lua/nvim-lsp-installer/servers/init.lua

```diff
@@ -31,72 +32,7 @@ local INSTALL_DIRS = {
     ["yamlls"] = "yaml",
 }
 
-local CORE_SERVERS = Data.set_of {
-    "angularls",
-    "ansiblels",
-    "bashls",
-    "bicep",
-    "clangd",
-    "clojure_lsp",
-    "cmake",
-    "cssls",
-    "denols",
-    "diagnosticls",
-    "dockerls",
-    "dotls",
-    "efm",
-    "elixirls",
-    "elmls",
-    "ember",
-    "emmet_ls",
-    "erlangls",
-    "eslint",
-    "eslintls",
-    "fortls",
-    "gopls",
-    "graphql",
-    "groovyls",
-    "hls",
-    "html",
-    "intelephense",
-    "jdtls",
-    "jedi_language_server",
-    "jsonls",
-    "kotlin_language_server",
-    "lemminx",
-    "ltex",
-    "ocamlls",
-    "omnisharp",
-    "phpactor",
-    "powershell_es",
-    "prismals",
-    "puppet",
-    "purescriptls",
-    "pylsp",
-    "pyright",
-    "rescriptls",
-    "rome",
-    "rust_analyzer",
-    "serve_d",
-    "solang",
-    "solargraph",
-    "sqlls",
-    "sqls",
-    "stylelint_lsp",
-    "sumneko_lua",
-    "svelte",
-    "tailwindcss",
-    "terraformls",
-    "texlab",
-    "tflint",
-    "tsserver",
-    "vala_ls",
-    "vimls",
-    "volar",
-    "vuels",
-    "yamlls",
-    "zls",
-}
+local CORE_SERVERS = Data.set_of(vim.tbl_keys(servers_metadata))
```

Comment:
This looks interesting, but how will it pick up new servers? Won't it just be permanently based off of what the last hard-coded list of `CORE_SERVERS` was?

Reply:
Yes, but the original list was always hard-coded anyway, but now it lives somewhere else. While with this change, you can get the list of supported servers (somehow) from lspconfig and cross-reference it against what's available. It could also be a scheduled CI job that takes care of that.

This already makes it easier to maintain, and makes sure you only have one source of truth.